js/shopping-list-item.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+class ShoppingListItem {
+  constructor(name, description) {
+    this.name = name;
+    this.description = description;
+    this.isDone = false;
+  }
+
+  check() {
+    return (this.isDone = true);
+  }
+
+  uncheck() {
+    return (this.isDone = false);
+  }
+
+  render(idx) {
+    if (this.isDone === true) {
```

Comment:
You could use the template string here such that there's always just one string that you're returning but just fill in the parts that you need to based on the `this.isDone` especially for the `checked` and `completed` parts. 

Also, on line 20, the `class` is misspelled for the first div.